CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# ChangeLog
+
+### [2023-03-03](https://github.com/go-enry/enry/pull/12)
```

Comment:
I would suggest not to host a copy of https://github.com/go-enry/enry/pull here, as it adds maintenance burden without much value.

IMO changelog would make some sense if it records the release history, but again only if the advantages overweight the burden on maintainers.

Reply:
Makes sense. You guys have been maintaining the repo for a while now. What do you suggest? If it's worth adding, then let's go with it, if not, it's not required. Also, if there is already some sort of note that keeps track of things on every version bump, then maintaining CHANGELOG could be redundant.

Reply:
I also agree on not adding changelog. I don't see much value here as most value anyway are in library not in tool itself.

Reply:
Copy that. I have now removed the CHANGELOG file.